src/plugins/discover/public/application/main/components/top_nav/on_save_search.tsx

```diff
@@ -124,13 +130,15 @@ export async function onSaveSearch({
       copyOnSave: newCopyOnSave,
       isTitleDuplicateConfirmed,
     };
+    const navigateOrReloadSavedSearch = !Boolean(onSaveCb);
```

Comment:
just for my understanding, onSaveBb is undefined if I switch away from SQL mode with saving the saved search, but not navigating to the new saved search, right?

Reply:
Correct! Is for the modal (Switch with saving first)

Reply:
thx 👍